File: Latihan_b/Python/Ram.py
class Ram:

    # atribut
    __capacity = ""
    __price = 0

    
    # konstruktor
    def __init__(self, capacity, price):
        self.__capacity = capacity
        self.__price = price

    # setter
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    # getter
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price

File: Latihan_b/Java/Ram.java
class Ram {

    // atribut
    private String capacity;
    private int price;

    
    // konstruktor
    public Ram(){}

    public Ram(String capacity, int price)
    {
        this.capacity = capacity;
        this.price = price;
    }

    // setter
    public void setCapacity(String capacity)
    {
        this.capacity = capacity;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // getter
    public String getCapacity()
    {
        return this.capacity;
    }

    public int getPrice()
    {
        return this.price;
    }
}